internal/presenters/presenter_sarif_results_pretty_test.go

```diff
@@ -76,8 +76,40 @@ func TestPresenterSarifResultsPretty_MediumHighIssuesWithColor(t *testing.T) {
 	require.Nil(t, err)
 
 	lipgloss.SetColorProfile(termenv.TrueColor)
-	result, err := presenters.PresenterSarifResultsPretty(input, testMeta)
+	result, err := presenters.PresenterSarifResultsPretty(input, testMeta, false, true)
 
 	require.Nil(t, err)
 	snaps.MatchSnapshot(t, result)
 }
+
+func TestPresenterSarifResultsPretty_DefaultHideIgnored(t *testing.T) {
```

Comment:
Suggestion: there is no test case for `true, false`

Reply:
Yes this case is currently covered by a workflow level test so wasn't convinced we wanted duplicate test coverage.